src/app/components/side-panel.js

```diff
@@ -29,8 +31,9 @@ const css = csjs`
     cursor: pointer;
   }
   .pluginsContainer {
-    height: calc(100% - 35px);
-    overflow: auto;
+    height: 100%;
```

Comment:
remove height here. flex: 1 should be enough.

Reply:
no it's not